DOCS/man/options.rst

```diff
@@ -1577,16 +1577,22 @@ Video
     the window, while ``--video-zoom`` can zoom in or out arbitrary amounts, and
     also works with ``--video-unscaled``.
 
-``--video-aspect-override=<ratio|no|original>``
+``--video-aspect-override=<ratio|no>``
     Override video aspect ratio, in case aspect information is incorrect or
     missing in the file being played.
 
-    These values have special meaning:
+    This value has special meaning:
+
+    :no: use the method of the ``--video-aspect-method`` option (default)
+
+    These values have special meaning only when the compatibility script is
+    loaded (see note):
 
     :0:  disable aspect ratio handling, pretend the video has square pixels
-    :no: same as ``0``
-    :original: use the video stream or container aspect (default)
-    :-1: same as ``1`` (deprecated)
+         (deprecated, use
+         ``--video-aspect-override=no --video-aspect-method=ignore`` instead)
+    :-1: use ``--video-aspect-method=<bitstream|container>`` behavior,
+         which is the behavior of ``-1`` for mpv 0.39 and earlier (deprecated)
```

Comment:
I'm not sure this is necessary to describe. We deprecate those option and remove them from documentation to make users transition. Behind the scenes we try our best to provide backward compatibility, but no guarantees are given. As it is often with depreciation. ffmpeg does this all the time, when deprecated things slowly break and become not functional, even before they are removed from the api.

On thing is to provide compatibility to not "crash" user application, another is functionality, the later should be best-effort only.